### PR TITLE
Only include latest release notes on NuGet

### LINKED ISSUE #742: Fix release notes from NuGet package
The whole release notes is added and nuget.org doesn't hide it under a toggle. 
It also has a character limit as it was pointed out on Discord.

Ideally only include the last release notes by parsing the content. But just pulling the last N lines is good enough.

image

### PR DESCRIPTION
**Pull request by Mitch528**
Resolves #742.

This PR uses an MSBuild task from https://github.com/sillsdev/SIL.BuildTasks. A caveat to using this would be that it follows the conventions of https://keepachangelog.com/en/1.0.0/.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## 3.0.0-alpha.11
 
+### Changed
```
Comment (Tyrrrz) [inline CHANGELOG.md:5]: Is this required for the tool to work?

Comment (Mitch528) [reply CHANGELOG.md:5]: Yeah, it seems to require at least one subheading on the latest version, though it doesn't necessarily need to be that exact one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -104,9 +106,9 @@ We'd love to get feedback.
 
 ## 2.2.0-alpha
 
-Add nullable reference types support (Sentry, Sentry.Protocol) (#509) 
-fix: Use ASP.NET Core endpoint FQDN (#485) 
-feat: Add integration to TaskScheduler.UnobservedTaskException (#481) 
+Add nullable reference types support (Sentry, Sentry.Protocol) (#509)
```
Comment (bruno-garcia) [inline CHANGELOG.md:109]: I think you editor doesn't like the empty spaces, it changed all files you saved

File: CHANGELOG.md (hunk 1 above)
Comment (bruno-garcia) [reply CHANGELOG.md:5]: It pulls the first subheading?

If so what if we had a first heading `#Changelog` and the subheading becomes the version?

**Review by bruno-garcia — COMMENTED** [review]
Thanks for taking in this @Mitch528 !! ❤️ 

CI doesn't seem happy though

**Comment by bruno-garcia** [comment]
OH nevermind it's Danger that fails, missing a changelog :D

File: CHANGELOG.md (hunk 1 above)
Comment (Mitch528) [reply CHANGELOG.md:5]: @bruno-garcia The #Changelog part should be fine. It selects the version by using the regex `^## (.+)`. What I meant was that the latest version requires at least one heading within, for example:

```md
# Changelog
...

## 1.0.1

### Added:
- ...
- ...
```

File: CHANGELOG.md (hunk 2 above)
Comment (Mitch528) [reply CHANGELOG.md:109]: oops, will change that back.

File: CHANGELOG.md (hunk 1 above)
Comment (bruno-garcia) [reply CHANGELOG.md:5]: Got it. maybe a good reason to start breaking changes into ### Fixes, Feature, etc

**Review by bruno-garcia — APPROVED** [review]
🥳

**Comment by bruno-garcia** [comment]
#skip-changelog

**Pull request merged**